assets/airports/loww.json

```diff
@@ -590,244 +549,94 @@
-            "entryPoints": {
-                "PESAT": ["PESAT"]
-            },
-            "body": [["PESAT", "A60+|S250-"], ["WW991", "A70|S250-"], ["WW989", "A60+|S250-"], ["WW981", "A50-|S250-"], ["WW979", "A50-|S250-"], ["WW977", "A50-|S250-"], ["WW975", "A50-|S250-"], ["WW974", "A50-|S250-"], ["WW973", "A45|S250-"], ["WW972", "A40|S250-"], ["WW971", "A30-|S250-"], ["WW970", "A30-|S250-"], ["RW34", "A50-|S250-"]],
-            "draw": [["PESAT", "WW991", "WW989", "WW981", "WW979", "WW977", "WW975", "WW974", "WW973", "WW972", "WW971", "WW970"]]
-        },
-        "LANUX6W": {
-            "icao": "LANUX6W",
-            "name": "Lanux six whiskey arrival",
-            "entryPoints": {
-                "LANUX": ["_LANUX140000"]
-            },
-            "body": [["LANUX", "A150-|S330"], ["NERDU", "A60-|S250"]],
-            "draw": ["LANUX", "NERDU"]
-        },
-        "MASUR2W": {
-            "icao": "MASUR2W",
-            "name": "Masur two whiskey arrival",
-            "entryPoints": {
-                "MASUR": ["_MASUR272009"]
-            },
-            "body": [["MASUR", "A170-|S330"], ["TEMTA", "A60-|S250"], "NERDU"],
-            "draw": [["MASUR", "TEMTA", "NERDU"]]
-        },
-        "GAMLI5W": {
-            "icao": "GAMLI5W",
-            "name": "Gamli five whiskey arrival",
-            "entryPoints": {
-                "GAMLI": ["LIMRA"]
-            },
-            "body": [["GAMLI", "A60+|S330+"], ["BARUG", "A170-|S330"], ["WW450", "A90-|S250-"], ["BALAD", "A80-|S250"]],
-            "draw": [["GAMLI", "BARUG", "WW450", "BALAD"]]
-        },
-        "XANUT1W": {
-            "icao": "XANUT1W",
-            "name": "Xanut one whiskey arrival",
-            "entryPoints": {
-                "XANUT": ["_NIGSI202010"]
-            },
-            "body": [["NIGSI", "A180-|S330"], ["BALAD", "A80|S250"]],
-            "draw": [["NIGSI", "BALAD"]]
-        },
-        "NATEX1W": {
-            "icao": "NATEX1W",
-            "name": "Natex one whisky arrival",
-            "entryPoints": {
-                "NATEX": ["_NATEX080019"]
-            },
-            "body": [["NATEX", "A140-|S330-"], ["PESAT", "A60-|S250"]],
-            "draw": [["NATEX", "PESAT"]]
-        },
-        "MIKOV7W": {
-            "icao": "MIKOV7W",
-            "name": "Mikov seven whiskey arrival",
-            "entryPoints": {
-                "MIKOV": ["_MIKOV345020"]
-            },
-            "body": [["MIKOV", "A170-|S330"], ["MABOD", "A60-|S250"]],
-            "draw": [["MIKOV", "MABOD"]]
-        },
-        "REKLU1W": {
-            "icao": "REKLU1W",
-            "name": "Reklu one whiskey arrival",
-            "entryPoints": {
-                "REKLU": ["_REKLU262000"]
-            },
-            "body": [["REKLU", "A180|S250-"], ["MABOD", "A60-|S250"]],
-            "draw": [["REKLU", "MABOD"]]
+        "11": {
+          "icao": "11",
+          "name": "runway 11",
+          "entryPoints": {
```

Comment:
Well not the _old_ old system, but yes-- the system we've designed for, such as that used in the latest version of MDSD, for example.